src/interpolation/compute/pipe.ts

```diff
@@ -0,0 +1,60 @@
+import { Formatters } from '../../types'
+import defaultFormatters from './formatters'
+import dot from './dot'
+import { UnvalidFormatterError, UnknownFormatterError } from './errors';
+
+export default (
+  value: any,
+  transformation: string,
+  data: any,
+  options?: { formatters?: Formatters; fallback?: string }
+): any => {
+  const formatters: Formatters = { ...defaultFormatters, ...(options && options.formatters) }
+  const matches = transformation.match(/^(?<formatter>[^\(]+)(\((?<params>.+)?\))?/)
+
+  if (!matches || !matches.groups || !matches.groups.formatter) {
+    throw new UnvalidFormatterError(`${transformation} is not a valid formatter.`)
+  }
+
+  const formatterName = matches.groups.formatter.toLowerCase()
+  const formatter = formatters[formatterName]
+
+  if (!formatter) {
+    throw new UnknownFormatterError(`${formatterName} is not defined.`)
+  }
+
+  let typedParams = []
+  if (matches.groups.params) {
+    try {
+      typedParams = normalizeParams(matches.groups.params.split(','), data, options)
+    } catch (err) { // Ignore the formatter if unknown variable (w/o fallback) in params
+      return value
+    }
+  }
+
+  return formatter(value, ...typedParams)
+}
+
+const normalizeParams = function(untypedParams: string[], data: any, options?: { formatters?: Formatters; fallback?: string }) {
+  return untypedParams.map(e => e.trim())
+    .map(e => {
+      if (/^(\“.*\”|\".*\"|\‘.*\’|\'.*\')$/.test(e)) { // String
+        return e.slice(1, -1)
+      } else if (/^[-+]?([0-9]+|[0-9]+\.[0-9]*|[0-9]*\.[0-9]+)$/.test(e)) { // Number
+        return Number(e)
+      } else if (/^true|false$/.test(e)) { //Boolean
+        return Boolean(e)
+      } else { // Variable
+        try {
+          return dot(e, data)
+        } catch (err) {
+          if ((options && options.fallback) === undefined) {
+            throw err
+          } else {
+            return options && options.fallback
```

Comment:
Should the fallback be used everytime we can't find a variable, or only for the result of interpolation ?

Ex:
```
{ first_name | concat(unknownVar) }

fallback: "[error]"
concat: txt, txt2 => txt2 ? txt+txt2 : txt
```

Should the results be: "Thibaud" or "Thibaud[error]" ?